pkg/netstate/client_test.go

```diff
@@ -110,27 +114,37 @@ func TestNetStateClient(t *testing.T) {
 			},
 			InlineSegment: []byte("raisins"),
 		},
+		XApiKey: []byte("abc123"),
 	}
 
 	_, err = c.Put(ctx, &pr2)
 	if err != nil || status.Code(err) == codes.Internal {
 		t.Error("Failed to Put")
 	}
 
+	putRes2, err := c.Put(ctx, &fp2)
+	assert.NoError(t, err)
+
 	if mdb.timesCalled != 3 {
 		t.Error("Failed to call mockdb correct number of times")
 	}
 
 	// Test Server.Delete
 	delReq := pb.DeleteRequest{
 		Path: []byte("here/is/a/path"),
+		XApiKey: []byte("abc123"),
 	}
 
 	_, err = c.Delete(ctx, &delReq)
 	if err != nil || status.Code(err) == codes.Internal {
 		t.Error("Failed to delete")
 	}
 
+	delRes, err := c.Delete(ctx, &delReq)
+	if err != nil {
+		t.Error("Failed to delete file path")
+	}
+
```

Comment:
this was my oversight during the rebase but this whole chunk (lines 143-147) can be deleted because it already happens in lines 138-141.